docs/operation/troubleshooting.rst

```diff
@@ -0,0 +1,124 @@
+If during a MetalK8s installation you encounter a failure and the console
+output does not provide sufficient information in order to pin-point the cause
+of failure, then re-run the installation with verbose arguments(--verbose).
+
+.. parsed-literal::
+
+   root@bootstrap $ /srv/scality/metalk8s-|release|/bootstrap.sh --verbose
+
+Errors after restarting the Bootstrap node
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you restart the Bootstrap node and for some reason, some containers
+(especially the salt-master container) refuses to start, check and ensure that
+the **MetalK8s ISO** is mounted properly.
+
+Bootstrap fails and console log is unscrollable
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If during a MetalK8s installation, the Bootstrap process fails and the console
+output is unscrollable then you can consult the Bootstrap logs in
+``/var/log/metalk8s-bootstrap.log``.
+
+Account Administration Errors
++++++++++++++++++++++++++++++
+
+Forgot the MetalK8s GUI password
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you forget the MetalK8s GUI username and/or password combination,
+follow
+:ref:`this procedure <Administering-MetalK8s-GUI-Kubernetes-API-and-Salt-API>`
+to reset or change it.
+
+General Kubernetes Resource Errors
+++++++++++++++++++++++++++++++++++
+
+Pod status shows "CrashLoopBackOff"
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If after a MetalK8s installation, you notice some Pods are in a state of
+"CrashLoopBackOff", then it means pods are crashing because they start up then
+immediately exit, thus Kubernetes restarts them and the cycle continues.
+To get possible clues about this error, run the following commands and inspect
+the output.
+
+.. code-block:: shell
+
+   kubectl -n <namespace-name> describe pod <pod name>
+
+   kubectl -n <namespace-name> logs -p  <pod name>
+
+Persistent Volume Claim(PVC) stuck in "Pending" state
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If after provisioning a volume for a Pod say Prometheus and the PVC still
+hangs in a ``Pending`` state, then try checking the following.
+
+Check that the volumes have been provisioned and it's status is ``Ready``:
+
+.. code-block:: shell
+
+   kubectl describe volume <volume name>
+
+Check that a corresponding Persistent volume exist:
+
+.. code-block:: shell
+
+   kubectl get pv
+
+If no Persistent volume exist, then check that the storage operator is up
+and running.
+
+.. code-block:: shell
+
+   kubectl -n kube-system get pods
+
+Access to MetalK8s GUI fails with "undefined backend"
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If in the cause of using the MetalK8s GUI, you encounter an "undefined
+backend" error then perform the following checks.
+
+Check that the Ingress pods are running:
+
+.. code-block:: shell
+
+   kubectl -n metalk8s-ingress get pods
+
+Check the Ingress controller logs:
+
+.. code-block:: shell
+
+   kubectl -n metalk8s-ingress logs <nginx-ingress-pod-name>
```

Comment:
Since Metal UI runs behind the control-plane Ingress, it would make sense to:

- provide a query to list all relevant Pods (using a label selector, e.g. `-l release=nginx-ingress-control-plane`)
- then show how to read the logs using the target Pod's name